src/render_tasks/d3d12_raytracing_task.hpp

```diff
@@ -251,6 +252,13 @@ namespace wr
 					d3d12::SetRTShaderSRV(cmd_list, 0, COMPILATION_EVAL(rs_layout::GetHeapLoc(params::full_raytracing, params::FullRaytracingE::SKYBOX)), skybox_t);
 				}
 
+        // Get brdf lookup texture
+        if (scene_graph.m_skybox.has_value())
+        {
+          auto brdf_lut_text = static_cast<d3d12::TextureResource*>(n_render_system.m_brdf_lut.value().m_pool->GetTexture(n_render_system.m_brdf_lut.value().m_id));

```

Comment:
"brdf_lut_text" this name does not make sense, what I'm reading right now is "brdf lookup texture text" , pretty sure you are not referring to text.